CHANGES.md

```diff
@@ -20,6 +20,8 @@ Format of the entries must be.
 
 * Node and cluster checks are executed in parallel. (DCOS_OSS-2239)
 
+* Docker is now part of the DC/OS node check. (DCOS_OSS-3845)
```

Comment:
This changelog is intended to be in reverse chronological order, so new entries should go at the top of the list. See guidelines for adding new entries at the top of this file.

I think the message could be clearer. How about "Node checks verify Docker is installed."

Reply:
Got it! Thank you @branden. I'll be sure to add that now.